### PR TITLE
EZP-28403: Design improvements for Sections

### PR DESCRIPTION
**Pull request by mikadamczyk**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZP-28403
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

Improve design for Section create and edit in Admin Panel

screen shot 2017-12-06 at 9 24 00 am
screen shot 2017-12-06 at 9 23 50 am

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review
- [x] Add right sidebar with action buttons
- [x] Keep Main Menu active on these sub-actions

### CODE REVIEW

**Comment by lserwatka** [comment]
@webhdx could you check this code for menu?

**Comment by lserwatka** [comment]
@sunpietro & @dew326 could you review frontend part here?

File: src/bundle/Resources/public/scss/_general.scss [hunk 1]
```diff
@@ -39,7 +39,7 @@ body {
     height: auto;
 }
 
-button {
+button, a {
```
Comment (sunpietro) [inline src/bundle/Resources/public/scss/_general.scss]: Maybe we should remove the `button, a` part and make the `.btn` styles more generic? cc @dew326

Comment (dew326) [reply src/bundle/Resources/public/scss/_general.scss]: Good point, +1

**Comment by webhdx** [comment]
Looks good. We might think of separating menu builders into different namespaces as `Menu\` is getting really bloated by the Builders. But this is something we can do with another menu once we need to add one.

**Pull request merged**